runtime/doc/news.txt

```diff
@@ -396,7 +355,7 @@ The following new APIs and features were added.
 ==============================================================================
 CHANGED FEATURES                                                 *news-changed*
 
-The following changes to existing APIs or features add new behavior.
+These existing features changed their behavior.
```

Comment:
If "add new behavior" is no longer in this section's description, some part of this section may need to be moved to "NEW FEATURES".

Reply:
i'll check. "adding new behavior" which doesn't conflict with existing behavior is definitely a "new feature", not a "changed feature".